src/components/pages/use_of_funds/FundsDistributionAccordion.vue

```diff
@@ -1,60 +1,27 @@
 <template>
 	<div class="funds_distribution_accordion">
-		<div v-for="fundsItem in applicationOfFundsData"
-			:key="fundsItem.id"
-			:class="[
-				'funds_distribution_info_item',
-				activeInfo[fundsItem.id] ? 'active' : ''
-			]"
-		>
-			<div
+		<details v-for="fundsItem in applicationOfFundsData" :key="fundsItem.id" class="funds_distribution_info_item">
```

Comment:
I've tried to use this with VoiceOver both on Firefox and Safari and there is a difference: When I activate the item on Safari (VoiceOverKey+Space), it loses the focus: When I press "next item" (VoiceOverKey+Right) it jumps to the main headline.

Reply:
I took `role="button"` off the summary element and Safari treats it a little better now